src/styles/layout/index.md.njk

```diff
@@ -17,6 +17,23 @@ Don’t make assumptions about what devices people are using. Design for differe
 
 The default maximum page width is 1020px, but you can make it wider if your content requires it.
 
+## Common layouts
+
+### Row 1: Two-thirds
+
+{{ example({group: 'styles', item: 'layout', example: 'common-two-thirds', html: true, open: true}) }}
+
+### Two-thirds / One-third
+
+{{ example({group: 'styles', item: 'layout', example: 'common-two-thirds-one-third', html: true, open: true}) }}
+
+### Row 1: Two thirds <br>Row 2: Two-thirds / One-third
+
+{{ example({group: 'styles', item: 'layout', example: 'common-two-thirds-two-thirds-one-third', html: true, open: true}) }}
+
+
+If you want to build your layout from scratch, here’s an explaination of how the page wrappers and grid system works.
```

Comment:
This perhaps gets a bit lost on the page, should it be a heading or be preceded by a section break?

Reply:
Typo: explaination  / explanation